website/docs/components/form/super-select/partials/guidelines/overview.md

```diff
@@ -0,0 +1 @@
+SuperSelect is a form element that provides users with a way to select either single or multiple options from a list within. It is an enhanced version of the HDS Select component. It supports rich content within option items, as well as before and after list components or content.
```

Comment:
I know it's mentioned in the code docs, but does it make sense to mention that this is built on the PowerSelect in this overview? I only ask because most designers are somewhat familiar with the PowerSelect at this point, so it might not be a bad idea.

Reply:
+1

Reply:
Added to overview.